website/source/docs/drivers/external/singularity.html.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: "docs"
+page_title: "Drivers: Singularity"
+sidebar_current: "docs-drivers-community-Singularity"
+description: |-
+  The Singularity task driver is used to run application containers using Singularity.
+---
+
+# Singularity Driver
+
+Name: `Singularity`
+
+The `Singularity` driver provides an interface for using Singularity for running application
+containers. You can download the external Singularity driver [here][Singularity-driver].
+
+
+## Task Configuration
+
+```hcl
+task "lolcow" {
+  driver = "Singularity"
+
+  config {
+     // this example run an image from sylabs container library with the
+     // canonical example of lolcow
+     image = "library://sylabsed/examples/lolcow:latest"
+     // command can be run, exec or test
+     command = "run"
+  }
+}
+```
+
+The `Singularity` driver supports the following configuration in the job spec:
+
+* `image` - The Singularity image to run. It can be a local path or a supported URI.
+
+    ```hcl
+    config {
+      image = "library://sylabsed/examples/lolcow:latest"
+    }
+    ```
+
+* `verbose` - (Optional) Enables extra verbosity in the Singularity runtime logging.
+Defaults to `false`.
+
+    ```hcl
+    config {
+      verbosity = "false"
```

Comment:
the example shows `verbosity`, but the documentation says `verbose`. i didn't check the Singularity driver to see which was correct.

Reply:
looks like the code uses the config [`verbose`](https://github.com/sylabs/nomad-driver-singularity/blob/master/pkg/plugin/driver.go#L70), and expects it to be a boolean

Reply:
totally a typo :see_no_evil: , fixed